src/app/identity-status/router.ts

```diff
@@ -0,0 +1,8 @@
+import express from "express";
+import { proveIdentityStatusCallbackGet } from "./middleware";
+
+const router = express.Router();
+
+router.get("/prove-identity-status", proveIdentityStatusCallbackGet);
```

Comment:
We probably want to rename this (and the handler) for our purposes, maybe `app-vc-receipt-status` ?